luasrc/controller/unblockneteasemusic.lua

```diff
@@ -24,7 +24,7 @@ end
 
 function act_status()
 	local e = {}
-	e.running = luci.sys.call("ps -w |grep unblockneteasemusic |grep app.js |grep -v grep >/dev/null") == 0
+	e.running = luci.sys.call("busybox ps -w |grep unblockneteasemusi[c] |grep /app.js >/dev/null") == 0
```

Comment:
Adding busybox here is enough.

Reply:
> Adding busybox here is enough.

used the regex could cuased less command